package.json

```diff
@@ -139,5 +139,9 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.8.5",
+    "@testing-library/jest-dom": "^5.16.5"
```

Comment:
I did this too. 

all of the dependencies should be dev dependencies so cf.gov doesn't install anything we're using and only takes the minified bundles we generate.

Reply:
Oh, good to know. I'll fix that

Reply:
Updated in 07108f0